audit-ci.json

```diff
@@ -4,6 +4,8 @@
     "GHSA-w5p7-h5w8-2hfq",
     "GHSA-7p7h-4mm5-852v",
     "GHSA-pfrx-2q88-qq97",
-    "GHSA-9c47-m6qq-7p4h"
+    "GHSA-9c47-m6qq-7p4h",
+    "GHSA-rc47-6667-2j5j",
```

Comment:
Can these vulnerabilities be resolved with `npm audit fix` instead of us adding it to this file?